app/src/sharedTest/java/org/oppia/app/topic/revision/TopicRevisionFragmentTest.kt

```diff
@@ -147,6 +150,9 @@ class TopicRevisionFragmentTest {
     }
   }
 
+  //Test failing in other devices too. RecyclerView item count
+  //Expected: <3>
+  //Got: <12>
   @Test
   fun testTopicRevisionFragment_loadFragment_configurationChange_checkSpanCount_isThree() {
```

Comment:
This test is failing in other devices too. There are 12 items in the recyclerview but check was made for 3 item count only.